test/run

```diff
@@ -143,24 +158,32 @@ def print_counter(counter):
 
 
 def main():
+    args = get_args()
     logging.basicConfig(format="run-tests: %(message)s")
     test_dir = pathlib.Path(__file__).resolve().parent
     root = test_dir.parent
     litani = root / "litani"
+    output_dir = pathlib.Path(args.output_dir).resolve()
+
+    temp = output_dir
+    while not temp.exists():
```

Comment:
I don't understand what this is for?

I think if `output_dir` doesn't exist, we should just create it

Reply:
`litani --init --output-prefix` does that for us.

Also if we create dir here, `chdir` into that and run `litani --init`, it will again create the same directories.
so the end result can be - `/tmp/litani-results/tmp/litani-results/...`

Reply:
Ah right. That's because your changing into `temp` and then setting `--output-prefix` to `output_dir`. I just noticed also that if the user passes a string, that would be a relative directory, but the default value is an absolute path 🍄.

Could you try: get rid of the loop, `chdir` into `args.output_directory`, and then
```
--output-prefix .
--output-symlink latest
```